mwzeval/utils.py

```diff
@@ -192,3 +201,95 @@ def parse_state(turn):
         json.dump(states, f, indent=2)
 
     return references, states
+
+
+def load_multiwoz24():
+    def is_filled(slot_value: str) -> bool:
+        """Whether a slot value is filled."""
+        slot_value = slot_value.lower()
+        return slot_value and slot_value != "not mentioned" and slot_value != "none"
```

Comment:
I'm not too sure about `none` semantics in the corpus ... when PRing upstream, ask the maintainers their opinion on this? I presume that 22 only has active slots in the frames and the point of this function is to achieve the same?

Reply:
tag MultiWOZ 2.4 paper authors as well asking for their opinion (if you can find them on github)